extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -105,7 +105,8 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
 
   /** @override */
   isValidElement() {
-    return isGoogleAdsA4AValidEnvironment(this.win) && this.isAmpAdElement();
+    return !!this.element.getAttribute('data-ad-client') &&
```

Comment:
This seems redundant, since if there's no client it'll get filtered out by adsenseIsA4AEnabled?